src/contentmanagermodel.cpp

```diff
@@ -150,3 +150,21 @@ bool ContentManagerModel::hasChildren(const QModelIndex &parent) const
     return true;
 }
 
+bool ContentManagerModel::canFetchMore(const QModelIndex &parent) const
+{
+    if (parent.isValid())
+        return false;
+    return (zimCount < m_data.size());
+}
+
+void ContentManagerModel::fetchMore(const QModelIndex &parent)
+{
+    if (parent.isValid())
+        return;
+    int remainder = m_data.size() - zimCount;
+    int zimsToFetch = qMin(5, remainder);
```

Comment:
Just a small number to not fetch too many, is there a better limit?

Reply:
Ok. So now the question is "from what are we fetching".
If we are fetching from the local library, there is no "too many" book to fetch.
For the remote library, it depends of how we manage it. If we fetch all book in the opds request, it would be useless to limit here. If we fetch only what we display, this is good.

But there is no real problem to keep this limit. We can keep it for now and advance on the PR. (Please open a issue to remember we have to think about this at a time)